src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int DATA_TYPE_INDEX = 0;
+    private static final int DATA_VALUE_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileData = readFromFile(fromFileName);
+        String report = reportCreation(fileData);
+        writeToFile(toFileName, report);
+    }
+
+    private String readFromFile(String fileName) {
+        StringBuilder stringBuilder;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            stringBuilder = new StringBuilder();
+            while (bufferedReader.ready()) {
+                stringBuilder.append(bufferedReader.readLine()).append(System.lineSeparator());
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+        return stringBuilder.toString();
+    }
+
+    private String reportCreation(String fileData) {
```

Comment:
```suggestion
    private String createReport(String data) {
```
Why 'fileData'? It is not important for this method where data came from.